driver/build.rs

```diff
@@ -0,0 +1,19 @@
+use ethcontract_generate::Builder;
+use std::env;
+use std::path::Path;
+
+fn main() {
+    generate_contract("BatchExchange", "batch_exchange.rs");
+    generate_contract("SnappAuction", "snapp_auction.rs");
+}
+
+fn generate_contract(name: &str, out: &str) {
+    let artifact = format!("../dex-contracts/build/contracts/{}.json", name);
```

Comment:
Does rust operate well with relative paths like this?

Reply:
Build scripts are always run from the root of the crate. So for the driver crate that's `driver/`.

Reply:
Cool I just remembered that there was a standard practice in python development that using relative paths for import statements was some sort of faux-pas.

Reply:
Here its needed unless you want to get needlessly fancy about it. Also its super deterministic seeing as it will always be relative to the same directory (unless the `driver` sub-crate moves).